src/app/components/apps-view/apps-view.component.ts

```diff
@@ -404,4 +404,14 @@ export class AppsViewComponent implements OnInit {
     this.matDrawer.close();
     this.removeRowSelection();
   }
+
+  copyLinkToClipboard() {
+    const url = window.location.href.split('?')[0];
+    const el = document.createElement('textarea');
+    el.value = `${url}?partition=${this.partitionSelected}&queue=${this.leafQueueSelected}&applicationId=${this?.selectedRow?.applicationId}`;
+    document.body.appendChild(el);
+    el.select();
+    document.execCommand('copy');
```

Comment:
document.execCommand('copy') is marked as deprecated. 
How about change to use:
-     navigator.clipboard.writeText(XXX)